Documentation/operations/upgrade.rst

```diff
@@ -979,7 +1276,7 @@ New ConfigMap Options
     enable only those features which user has explicitly enabled in their
     ConfigMap. See :ref:`kubeproxy-free` for more option values.
 
-    For users who previously were running with ``nodePort.enabled=true`` it is
+    For users who previously were running with ``global.nodePort.enabled=true`` it is
```

Comment:
was this supposed to be modified?

Reply:
this should have the global prefix as this refers to users upgrading from 1.7x, so it seems like this was an error that never got caught